packages/cli/src/commands/login/index.ts

```diff
@@ -24,8 +24,11 @@ import { getFlagsSpecification } from '../../util/get-flags-specification';
 import { printError } from '../../util/error';
 import output from '../../output-manager';
 import { LoginTelemetryClient } from '../../util/telemetry/commands/login';
+import { login as future } from './future';
 
 export default async function login(client: Client): Promise<number> {
+  if (client.argv.slice(2).includes('--future')) return await future(client);
```

Comment:
If this is shifted down, `--future` will be available as `parsedArgs.flags['--future']`